### ISSUE TITLE
Failed to load model (from_pretrained): Dimension out of range (expected to be in range of [-1, 0], but got -2)`

### ISSUE THREAD

**Issue by fengcheche** [OpenBMB/VoxCPM#241]
win1 ，Python 3.11.9，32G内存，GPU 0

	NVIDIA GeForce RTX 4070 Laptop GPU

	驱动程序版本:	32.0.15.7283
	驱动程序日期:	2025/3/14
	DirectX 版本:	12 (FL 12.1)
	物理位置：	PCI 总线 1、设备 0、功能 0

	利用率	15%
	专用 GPU 内存	1.6/8.0 GB
	共享 GPU 内存	0.1/15.9 GB
	GPU 内存	1.7/23.9 GB

就跑了3个命令就报错了
pip install voxcpm
$env:HF_ENDPOINT = "https://hf-mirror.com"
voxcpm design     --text "Hello from VoxCPM!"  --output "E:\voxcpm\out.wav"

PS E:\voxcpm> voxcpm design     --text "Hello from VoxCPM!"  --output "E:\voxcpm\out.wav"
Loading VoxCPM model...
Fetching 8 files: 100%|██████████████████████████████████████████████████████████████████████████| 8/8 [00:00<?, ?it/s]
Download complete: : 0.00B [00:00, ?B/s]              voxcpm_model_path: C:\Users\Administrator\.cache\huggingface\hub\models--openbmb--VoxCPM2\snapshots\e8b928065859f2869644c1e2881cbd21f888c659, zipenhancer_model_path: None, enable_denoiser: True
Download complete: : 0.00B [00:00, ?B/s]
C:\Python\Python311\Lib\site-packages\torch\nn\utils\weight_norm.py:144: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
Loading AudioVAE from pytorch: C:\Users\Administrator\.cache\huggingface\hub\models--openbmb--VoxCPM2\snapshots\e8b928065859f2869644c1e2881cbd21f888c659\audiovae.pth
Running on device: cpu, dtype: bfloat16
Loading model from safetensors: C:\Users\Administrator\.cache\huggingface\hub\models--openbmb--VoxCPM2\snapshots\e8b928065859f2869644c1e2881cbd21f888c659\model.safetensors
Warning: torch.compile disabled - VoxCPMModel can only be optimized on CUDA device
Loaded VoxCPM2Model
Warm up VoxCPMModel...
  0%|                                                                                           | 0/10 [00:01<?, ?it/s]
Failed to load model (from_pretrained): Dimension out of range (expected to be in range of [-1, 0], but got -2)

**Comment by fengcheche** [comment]
问了cursor说好像是
Running on device: cpu, dtype: bfloat16

我有GPU，但是不知道为什么不在GPU上跑呢

**Comment by fengcheche** [comment]
重装了系统，好像是没装cuda和pytorch,我先去安装

**Comment by fengcheche** [comment]
确实装好就OK了
